make/autoconf/jvm-features.m4

```diff
@@ -44,7 +44,7 @@
 m4_define(jvm_features_valid, m4_normalize( \
     ifdef([custom_jvm_features_valid], custom_jvm_features_valid) \
     \
-    cds compiler1 compiler2 dtrace epsilongc g1gc jfr jni-check \
+    cds compiler1 compiler2 dtrace epsilongc g1gc hardlse jfr jni-check \
```

Comment:
The feature should be named `hard-lse` to match the `HARD_LSE` define in Hotspot code. (Also, it improves readability.)